integration/concurrency_test.go

```diff
@@ -106,7 +106,7 @@ func TestIntegrationConcurrentPredictOrcaMini(t *testing.T) {
 
 // Stress the system if we know how much VRAM it has, and attempt to load more models than will fit
 func TestMultiModelStress(t *testing.T) {
-	vram := os.Getenv("OLLAMA_MAX_VRAM")
+	vram := os.Getenv("OLLAMA_MAX_VRAM") // TODO - discover actual VRAM
```

Comment:
Did you mean to add this (if the env var is deprecated?)

Reply:
We still rely on this in the tests because there isn't an API to expose GPU VRAM.